### PR TITLE
Add `Has` method

### LINKED ISSUE #99: Add a method to check if a key exists
The current way to check if a key exists is to use Get and check for nil. While it works it create boilerplate code.

It would be most convenient if there was a method to check if a key exists. For instance:

```
if cache.Has("key123") {
    return cache.Get("key123")
}
```

Thanks

### PR DESCRIPTION
**Pull request by gozeloglu**
* `Has` method is implemented and tested.
* This change fixes #99.

### CODE REVIEW

File: cache_test.go [hunk 1]
```diff
@@ -599,6 +599,42 @@ func Test_Cache_Delete(t *testing.T) {
 	assert.NotContains(t, cache.items.values, "1")
 }
 
+func TestCache_Has(t *testing.T) {
```
Comment (swithek) [inline cache_test.go:602]: `TestCache_Has` -> `Test_Cache_Has`

File: cache_test.go [hunk 2]
```diff
@@ -599,6 +599,42 @@ func Test_Cache_Delete(t *testing.T) {
...
+	testCases := []struct {
```
Comment (swithek) [inline cache_test.go:603]: For consistency, can you turn these into map-based tests?

Comment (swithek) [reply cache_test.go:603]: like in `Test_Cache_Get`

**Review by swithek — COMMENTED** [review]
Looks good. A few small changes are needed.

**Review by swithek — APPROVED** [review]
Thanks for the PR.

**Comment by lesichkovm** [comment]
Thank you @gozeloglu

**Pull request merged**